src/AgDatabase.cs

```diff
@@ -25,7 +25,7 @@ public interface IAgDatabase
     List<BackupMetadata> RecentBackups();
     void JoinAg();
 
-    void Restore(IEnumerable<BackupMetadata> backupOrder, Func<string, string> fileRelocation = null);
+    void Restore(IList<BackupMetadata> backupOrder, Func<string, string> fileRelocation = null);

```

Comment:
I don't see any reason for this change. We don't need to add any items to the list as a part of the restore so IEnumerable will suffice and more accurately portray the intentions.